flocker/provision/_aws.py

```diff
@@ -36,6 +36,7 @@
     'centos-7': 'centos',
     'ubuntu-14.04': 'ubuntu',
     'ubuntu-15.10': 'ubuntu',
+    'rhel-7.2': 'ec2-user',
```

Comment:
There are a lot of places which we need to change to support a new distribution. We should probably localize them into the [Strategy Pattern](https://en.wikipedia.org/wiki/Strategy_pattern)